command/exec.go

```diff
@@ -161,10 +168,15 @@ func (c *ExecCommand) Run(args []string) int {
 			return 1
 		}
 		c.conf.script = buf.Bytes()
+	} else {
+		if !c.conf.shell {
```

Comment:
this can be `else if !c.conf.shell {` on line 171, I think

Reply:
fixed